configure.in

```diff
@@ -476,6 +476,25 @@ WARNING :
   AC_MSG_RESULT(no)
 )
 
+max_fcrontab_load_delay_seconds=60
+AC_MSG_CHECKING(max fcrontab load delay seconds)
+AC_ARG_WITH(max-fcrontab-load-delay-seconds,
+[ --with-max-fcrontab-load-delay-seconds=INTEGER	The maximum delay, in seconds, before loading the fcrontab as any user (0 for no delay, like root user)],
```

Comment:
- hmm, change-delay may also be inaccurate, since the fcrontab is not necessarily being changed.
maybe reload? that should always be true, right?
i.e.: `--with-max-fcrontab-reload-delay-seconds`

- well, max-delay does not imply any sort of min delay (and in fact the actual delay can always be less then the min-delay, unless --max=1s).

Reply:
"reload" sounds good, thanks.

As for the second part, sorry I possibly wasn't clear. I wasn't talking about a minimum here, but the fact that setting `--with-max-fcrontab-reload-delay-seconds` to more than 60 would be exactly the same as setting it to 60, so 60 is effectively the max meaningful value for `--with-max-fcrontab-reload-delay-seconds`.
This is because I think `delay_s` can never be higher than 60s given line 111 to 115, even if `--with-max-fcrontab-reload-delay-seconds` was set to a bigger value like say 300 ?
I am not sure why someone would try setting it to a bigger value than 60, but if they did they may be surprised that the delay never gets more than 60s? 

So the description could become: `The maximum delay, in seconds, before (re)loading the fcrontab as any user (0 for no delay, like root user). Note that if set to a value higher than 60, the maximum delay will be 60s nonetheless.`

Reply:
cool.
heh, yea, i was saying that since setting a ceiling does not make any guarantees about a floor, there is no need to mention that the actual value may be smaller than the max, even if it will ALWAYS be smaller for some values.

however, i agree that the behavior is complicated and bears explaining, so i wrote a novel, a sequel, and a coffee table companion guide:

The maximum delay, in seconds, before (re)loading the fcrontab as a non-root user (there is never any delay for root). By default, the delay to load the fcrontab for a non-root user is at least one full second and at most sixty seconds, depending on circumstances when the load happens. A value of 0 for this argument means no delay for any user, ever. A value between 1 and 59 (inclusive) will ensure that the non-root delay is always at least 1s and never more than the given number of seconds. A value greater than or equal to 60 has no effect.